scripts/run_agent.sh

```diff
@@ -81,7 +79,8 @@ if [ "$AGENT_ENVIRONMENT" = "preview" ] && [ -n "$PR_NUMBER" ]; then
             npm run agent:build
             screen -dmS agent-preview bash -c 'npm run agent:start > out.log 2> error.log'
         else
```

Comment:
I would just simplify both this and the above if statements to always assume these directories exist